app/code/Magento/PageBuilder/view/adminhtml/web/js/content-type/banner/converter/style/overlay-background-color.js

```diff
@@ -2,8 +2,8 @@
 /* jscs:disable */
 define(["Magento_PageBuilder/js/utils/object"], function (_object) {
   /**
-   * Copyright © Magento, Inc. All rights reserved.
-   * See COPYING.txt for license details.
+   * Copyright 2025 Adobe
```

Comment:
Copyright year should be the year when the file was created:
```suggestion
   * Copyright 2018 Adobe
```

Reply:
Hello @engcom-Hotel 
Changed the copyright year to reflect the file's creation year.